### ISSUE TITLE
Can I not load velocity.min.js and velocity.ui.min.js?

### ISSUE THREAD

**Issue by webees** [theme-next/hexo-theme-next#202]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 

After all animations are disabled, velocity.min.js and velocity.ui.min.js should not be loaded.

(Partial animation can be done through jquery.js. Instead of relying on velocity.js.)

### Actual behavior 

Utils.js and motion.js have dependency on velocity.js. It seems that some animations cannot be closed by configuration.

### Steps to reproduce the behavior 
1. motion 》》enable: false

#### Hexo version

```
hexo: 3.6.0
hexo-cli: 1.1.0
os: Windows_NT 10.0.16299 win32 ia32
http_parser: 2.7.0
node: 8.7.0
v8: 6.1.534.42
uv: 1.15.0
zlib: 1.2.11
ares: 1.10.1-DEV
modules: 57
nghttp2: 1.25.0
openssl: 1.0.2l
icu: 59.1
unicode: 9.0
cldr: 31.0.1
tz: 2017b
```

### NexT Information

**NexT Version:**

- [ ] Latest Master branch
- [x] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [x] All schemes
- [ ] Muse
- [ ] Mist
- [ ] Pisces
- [ ] Gemini

#### NexT Configuration:

```yml
motion:
  enable: false
```

**Comment by sli1989** [comment]
The PR #113 has discussed about the issue. Waiting for optimization.

**Comment by webees** [comment]
good job!

**Comment by stevenjoezhang** [comment]
Fixed in #951
